data/json/monsters/mechsuits.json

```diff
@@ -0,0 +1,139 @@
+[
+  {
+    "id": "mon_mech_recon",
+    "type": "MONSTER",
+    "name": "X-03: 'Spectre' recon mech",
+    "description": "The Boeing-Daewoo RMES ( Recon Mechanical Exoskeleton Suit ), a recent acquisition by the US military, it was designed to be used in a scout-recon-sniper role, due to it's mobility and integrated laser sniper rifle and suite of optics for target designation and battlefield awareness.  It was not deployed before the Cataclsym hit, though there were a few prototypes in the field.  You may be able to hack it to accept you as it's pilot.  Like all mech-suits it can act as a UPS from it's large battery.",
```

Comment:
it's -> its (all 3 instances here) and other descriptions with similar phrasing (mon_mech_combat, mon_mech_lifter).

Reply:
You think they should all have different phrasing? I think there are other things that are multiple examples of a type of item, that have similar descriptions.

Reply:
O I see, the it's its again